content/courses/onchain-development/intro-to-anchor.md

```diff
@@ -90,13 +105,20 @@ mod program_module_name {
 }
```

Comment:
Formatting above looks odd, see CONTRIUBUTING.md

Reply:
I think it only appears like this in the diff window. I checked locally and on my github branch, its appears to be properly formatted there.

Reply:
![image](https://github.com/user-attachments/assets/8dd0bb49-aaa4-4508-8183-21e492a12f89)

Reply:
I bet it's using a mix of tabs and spaces, and your local branch probably has some value set for tabs (2 spaces, 4 spaces, 8 spaces) that differs from what github uses. So yes it's a prob - please do fix it!

Reply:
Can you show/tell me whats exactly out of place here, I didn't touch this part and it seems like it is formatted like everything Rust part in the file. Also used `rustfmt` as instructed in CONTRIBUTING.md, nothing changes.